integration/spark/shared/src/main/java/io/openlineage/spark/agent/lifecycle/SparkOpenLineageExtensionVisitorWrapper.java

```diff
@@ -190,10 +199,28 @@ private Optional<ImmutablePair<Object, Method>> getMethod(
   }
 
   private static List<Object> init(String testExtensionProvider)
```

Comment:
Isn't it that `init` method is created anytime OpenLineage context is created and each time `ByteBuffer` with class bytes is read?

Reply:
done